salt/states/consul.py

```diff
@@ -0,0 +1,169 @@
+# -*- coding: utf-8 -*-
+'''
+Consul Managemnt
+===========================
+
+The consul module is used to create and manage ACL
+
+.. code-block:: yaml
+
+    acl_present:
+      consul.acl_present:
+        - id: 38AC8470-4A83-4140-8DFD-F924CD32917F
+        - name: acl_name
+        - rules: node "" {policy = "write"} service "" {policy = "read"} key "_rexec" {policy = "write"}
+        - type: client
+        - consul_url: http://localhost:8500
+
+    acl_delete:
+       consul.acl_absent:
+         - id: 38AC8470-4A83-4140-8DFD-F924CD32917F
+'''
+
+
+def _acl_changes(name, id=None, type=None, rules=None, consul_url=None, token=None):
+    '''
+       return True if the acl need to be update, False if it doesn't need to be update
+    '''
+    r = __salt__['consul.acl_info'](id=id, token=token, consul_url=consul_url)
```

Comment:
I know this is picky, but can you change these variable references to be at least 3 letters? It's a stylistic standard we like to have at Salt. `ret` is fine.

This needs to be changed in some of the other functions below as well.